requirements-testing.txt

```diff
@@ -1,10 +1,11 @@
-black == 24.*
 coverage[toml] == 7.*
-moto[s3,sts] == 5.*
-mypy == 1.*
 pytest == 8.*
-pytest-cov == 5.*
+pytest-cov == 6.*
 pytest-xdist == 3.*
-ruff == 0.7.*
-twine == 5.*
+twine == 6.*
+mypy == 1.*
+black == 24.*
+ruff == 0.8.*
+moto[s3,sts] == 5.*
+pipgrip == 0.10.*
```

Comment:
This looks like a new dependency, do we need it?

Reply:
Good catch. I copy the dependencies list from `deadline-cloud-for-maya` to have them in the same order. Didn't notice this extra one. Will remove it.